### PR TITLE
feature: home page banner get and patch endpoints. TM-913

### PR DESCRIPTION
**Pull request by rtirserio**
Completed TM-913 by adding endpoints for the home page banner.

Running the migration will create the one entry for the banner. There is no reason for there to be more than one entry in the banner table.

### CODE REVIEW

**Comment by mjoyce91** [comment]
- Can you add authorization? I was able to to patch without being logged in

- Was also forced to provide all fields in the PATCH (`text` and `is_visible`) - can it be changed so that you don't have to provide every field?

- Can the swagger endpoint be updated to display the body input field?
Screen Shot 2019-06-05 at 9 31 23 AM
ex: PATCH /position/{id} looks like:
Screen Shot 2019-06-05 at 9 32 11 AM

File: talentmap_api/administration/views/homepage.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+from talentmap_api.common.permissions import isDjangoGroupMemberOrReadOnly
+
+from talentmap_api.administration.models import HomepageBanner
+from talentmap_api.administration.serializers.homepage import HomepageBannerSerializer
+
+class HomepageBannerView(mixins.RetrieveModelMixin, mixins.UpdateModelMixin, GenericViewSet, APIView):
+
+    serializer_class = HomepageBannerSerializer
+    # permission_classes = (IsAuthenticatedOrReadOnly, isDjangoGroupMemberOrReadOnly('superuser'))
```
Comment (mjoyce91) [inline talentmap_api/administration/views/homepage.py]: Looks like this addresses my comment

**Comment by rtirserio** [comment]
@mjoyce91 comments addressed

**Comment by rtirserio** [comment]
@mjoyce91 yes, about page will look pretty much just like this.

**Pull request merged**